### PR TITLE
[NAT-2020] - Add DevOpsDays Natal 2020

### PR DESCRIPTION
**Pull request by Igorolivei**
This Pull Request adds the DevOpsDays Natal 2020.

The e-mail with the organizers list has already been sent.

### CODE REVIEW

File: content/events/2020-natal/patrocinio.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Oportunidade de exibição do logotipo da empresa no site, cartazes e banners - Tamanho: ¾U
+
+6 Inscrições para o evento
+
+Oportunidade de receber agradecimento formal no início de cada sessão de apresentação
+
+Estande
+
+
+BONZE:
```
Comment (iennae) [inline content/events/2020-natal/patrocinio.md]: Is this supposed to be BRONZE?

File: data/events/2020-natal.yml [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  - name: "Leonardo Ataide Minora"
+    role: "Organizer"
+    bio: ""
+    linkedin: "https://www.linkedin.com/in/leominora/"
+    website: "http://diatinf.ifrn.edu.br/doku.php?id=pessoal:docente:efetivo:minora"
+    github: "https://github.com/leonardo-minora/"
+    twitter: "@leonardominora"
+    image: "leonardo-minora.jpg"
+
+organizer_email: "organizers-natal-2020@devopsdays.org" # Put your organizer email address here
```
Comment (iennae) [inline data/events/2020-natal.yml]: I think this should just be the city versus with the year.

Comment (Igorolivei) [reply data/events/2020-natal.yml]: Hey @iennae , has the pattern changed? This year for Natal and Campinas was organizers-<city>-<year>@devopsdays.org

File: content/events/2020-natal/patrocinio.md (hunk 1 above)
Comment (Igorolivei) [reply content/events/2020-natal/patrocinio.md]: Yes, you're right! Thanks for noticing!

File: data/events/2020-natal.yml (hunk 2 above)
Comment (bridgetkromhout) [reply data/events/2020-natal.yml]: Yes, it’s changed for 2020 and beyond. It’s simpler and doesn’t need to be changed every year.

**Comment by bridgetkromhout** [comment]
Note to reviewers: ensure email/Slack is updated before merging.

File: data/events/2020-natal.yml (hunk 2 above)
Comment (Igorolivei) [reply data/events/2020-natal.yml]: Nice! Thanks!

**Comment by bridgetkromhout** [comment]
Hi, @Igorolivei -

1) You are listing all the same speakers as last year. Are you sure you want to do that?
2) We do not allow you to announce dates until you have your location set. 

I have made the necessary changes for Slack and email, but we cannot merge this PR with the dates listed since you don't have a location, and I do not think you intended to list all the same speakers as last year. Please fix those two things.

@gomex @somatorio please explain - thanks.

**Comment by Igorolivei** [comment]
Hi @bridgetkromhout , thanks for noticing the speakers issue.

I didn't know that I need to have the location set to announce the dates, so I closed the PR and I'll reopen when this is fixed.

Thanks!

**Pull request closed without merging**